src/modules/home/components/single-property/single-property-review.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Rating from '@mui/material/Rating';
+import StarIcon from '@mui/icons-material/Star';
+import Button from '@mui/material/Button';
+import Divider from '@mui/material/Divider';
+import { SinglePropertyReviewStyles } from './single-property-review.styles';
+
+interface Review {
+  id: string;
+  name: string;
+  comment: string;
+  rating: number;
+}
+
+interface SinglePropertyReviewProps {
+  reviews: Review[];
+}
+
+function SinglePropertyReview({ reviews }: SinglePropertyReviewProps) {
```

Comment:
Same problem here. SinglePropertyReview is giving off the thought that this is a component that displays a single review, but actually it's mapping several reviews. So name this SinglePropertyReviews.